storage/src/main/java/org/apache/kafka/storage/internals/log/UnifiedLog.java

```diff
@@ -40,6 +47,16 @@ public class UnifiedLog {
 
     private static final Logger LOG = LoggerFactory.getLogger(UnifiedLog.class);
 
+    public static final String LOG_FILE_SUFFIX = LogFileUtils.LOG_FILE_SUFFIX;
```

Comment:
Similarly, since `LogFileUtils` and `UnifiedLog` are in the same package, could we just keep one constant reference?

Reply:
I kept those for the same reason as described in https://github.com/apache/kafka/pull/18039#discussion_r1886615158